messaging/schema.py

```diff
@@ -327,7 +359,15 @@ class Input:
     @map_enums_to_values_in_kwargs
     def mutate_and_get_payload(cls, root, info, **kwargs):
         message = get_obj_if_user_can_administer(info, kwargs["id"], Message)
-
+        # user = info.context.user
+
+        # if validate_recipient_selection_and_data(
+        #     message.recipient_selection,
+        #     message.event,
+        #     message.occurrences,
+        #     user,
+        #     message.project,
+        # ):
```

Comment:
Why is there commented out code here?

Reply:
Should have had it as a draft or something. There is now an actual permission check there. I was jsut waiting that I'll test the feature with admin UI first.